### ISSUE TITLE
Restrict datasets shown in Maps page.

### ISSUE THREAD

**Issue by ldesousa** [ERIN-LIST/pyguess#28]
The Maps page can only present to the User the Datasets contained in the Workspaces (s)he has access to.

Depends on #18 and #25.

**Comment by uleopold** [comment]
We can put this for now on hold as we will probably stil need to adapt the concept of workspaces.
